irc/ircii/Portfile

```diff
@@ -36,6 +36,17 @@ configure.env       COPY_DIRECTORY=tar
 configure.args      --enable-ipv6 \
                     --with-default-server=irc.freenode.net
 
+# translat.h:24: error: redefinition of typedef ‘libiconv_t’
+compiler.blacklist-append \
+                    *gcc-4.0 *gcc-4.2
+
+# help.c:535:50: error: passing argument 3 of 'scandir'
+# from incompatible pointer type [-Wincompatible-pointer-types]
+if {[string match *gcc* ${configure.compiler}]} {
+    configure.cflags-append \
+                    -Wno-error=incompatible-pointer-types
```

Comment:
As usual, I note that upstream probably wants to know about such problems.

Reply:
Homepage returns “The page you requested could not be found”.

Reply:
Try talking to mrg a t netbsd.org

Reply:
The current homepage might be http://eterna23.net/ircii/

Reply:
“ircII comes with a script called ircbug that should be used to send bug reports”

Sorry, I am not gonna do this. If everyone prefers having this port broken, so be it, it is not important for me. There is more important stuff in the pipeline to complete.